sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogSuite.scala

```diff
@@ -46,4 +46,23 @@ class HiveExternalCatalogSuite extends CatalogTestCases {
 
   protected override def resetState(): Unit = client.reset()
 
+  import utils._
+
+  test("drop database cascade with function defined") {
```

Comment:
I think this test case should be in `ExternalCatalogSuite`. Both `InMemoryCatalog` and `HiveExternalCatalog` have the same behavior if `cascade` is on.